conf/deployment/aws/ipi_3az_rhcos_3m_3w_sts.yaml

```diff
@@ -0,0 +1,20 @@
+---
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
+  sts_enabled: true
```

Comment:
Document in `conf/README.md`

Reply:
Add also ccoctl_version please

Reply:
> Document in `conf/README.md`

Will do

> Add also ccoctl_version please

We aren't using ccoctl_version for this deployment type, just ibmcloud. I have created a ticket here to potentially unify this for both platforms: https://url.corp.redhat.com/d888415